### PR TITLE
Enable Brotli compression for RPC requests by default

### PR DESCRIPTION
**Pull request by GytisCepk**
### Motivation
Enabling compression for RPC response bodies have show to significantly drop size of data which needs to be downloaded by the frontend. Right now, only certain RPC requests have Gzip compression enabled.

Also, Brotli compression have shown to have better compression ratio for our RPC requests without additional compressing time.

### Changes
- Make Brotli main compression instead of Gzip (if Brotli not supported by the browser, will fallback to Gzip).
- Enable compression for all RPC responses by default (to disable compression for specific RPC request, add following line above definition: `@RpcOperation.setPolicy({ allowResponseCompression: false })`.

### CODE REVIEW

File: core/backend/src/rpc/web/response.ts [hunk 1]
```diff
@@ -24,6 +24,10 @@ import {
   WebAppRpcRequest,
 } from "@itwin/core-common";
 
+import { Readable, Stream } from "stream";
```
Comment (aruniverse) [inline core/backend/src/rpc/web/response.ts:27]: could you prefix all node api imports with the node protocol?

eg

```suggestion
import { Readable, Stream } from "node:stream";
```

[the remark this answers is not in the document]
Comment (GytisCepk) [reply core/backend/src/rpc/web/response.ts:27]: If "dta test app" is display-test-app, test app works fine.

This is backend code so it shouldn't affect frontend. Dynamic imports, I believe, are leftovers from when this code was in core-common package ([commit where this was moved to core-backend](https://github.com/iTwin/itwinjs-core/commit/bd70948675a9cfaccba6f199dc331239e0c3b524#diff-0d2292b2aa47df14713978458cdf9705ded4a92eb9425091732276e9f556d5cd)).

Comment (GytisCepk) [reply core/backend/src/rpc/web/response.ts]: Prefixed all new Node imports in this PR

**Review by ben-polinsky — APPROVED** [review]

File: core/backend/src/rpc/web/response.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (aruniverse) [reply core/backend/src/rpc/web/response.ts]: ah, i didnt even realize it moved to core-backend, it should be ok then. thx for sharing

**Review by aruniverse — APPROVED** [review]

**Review by saskliutas — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Review by grigasp — APPROVED** [review]

**Pull request merged**